addons/skin.estuary/xml/DialogPVRInfo.xml

```diff
@@ -1,126 +1,365 @@
+						<visible>!String.IsEmpty(ListItem.ChannelLogo) + !String.IsEmpty(ListItem.Art(thumb)) + String.IsEmpty(ListItem.EpgEventIcon)</visible>
+					</control>
+				</control>
 			</control>
 			<control type="group">
-				<top>230</top>
-				<left>50</left>
 				<include>OpenClose_Right</include>
-				<control type="button" id="61">
-					<left>600</left>
-					<top>-18</top>
-					<width>1170</width>
-					<height>509</height>
-					<font></font>
-					<onleft>61</onleft>
-					<onright>61</onright>
-					<ondown>9000</ondown>
-					<onup>9000</onup>
-					<onclick>SetProperty(TextViewer_Header,$LOCALIZE[207],home)</onclick>
-					<onclick>SetProperty(TextViewer_Text,$ESCINFO[ListItem.Plot],home)</onclick>
-					<onclick>ActivateWindow(1102)</onclick>
+				<left>600</left>
+				<top>-30</top>
+				<control type="group" id="140">
+					<top>164</top>
+					<control type="button" id="138">
+						<width>774</width>
+						<height>718</height>
+						<textoffsetx>40</textoffsetx>
+						<textoffsety>20</textoffsety>
+						<aligny>bottom</aligny>
+						<label></label>
+						<onclick condition="!String.IsEmpty(ListItem.Plot)">SetProperty(TextViewer_Header,$LOCALIZE[207],home)</onclick>
+						<onclick condition="!String.IsEmpty(ListItem.Plot)">SetProperty(TextViewer_Text,$ESCINFO[ListItem.Plot],home)</onclick>
+						<onclick condition="!String.IsEmpty(ListItem.Plot)">ActivateWindow(1102)</onclick>
+						<onup>5000</onup>
+						<onleft>4000</onleft>
+						<onright>4000</onright>
+						<ondown>5000</ondown>
+						<texturenofocus border="21">dialogs/dialog-bg.png</texturenofocus>
+						<visible>Integer.IsGreater(Container(4000).NumItems,0)</visible>
+					</control>
+					<control type="image">
+						<left>755</left>
+						<width>512</width>
+						<height>718</height>
+						<aligny>bottom</aligny>
+						<texture border="21">dialogs/dialog-bg.png</texture>
+						<visible>Integer.IsGreater(Container(4000).NumItems,0)</visible>
+					</control>
+					<control type="button" id="139">
+						<width>1265</width>
+						<height>718</height>
+						<textoffsetx>40</textoffsetx>
+						<textoffsety>20</textoffsety>
+						<aligny>bottom</aligny>
+						<label></label>
+						<onclick condition="!String.IsEmpty(ListItem.Plot)">SetProperty(TextViewer_Header,$LOCALIZE[207],home)</onclick>
+						<onclick condition="!String.IsEmpty(ListItem.Plot)">SetProperty(TextViewer_Text,$ESCINFO[ListItem.Plot],home)</onclick>
+						<onclick condition="!String.IsEmpty(ListItem.Plot)">ActivateWindow(1102)</onclick>
+						<onup>50</onup>
+						<onleft>4000</onleft>
+						<onright>4000</onright>
+						<ondown>5000</ondown>
+						<texturenofocus border="21">dialogs/dialog-bg.png</texturenofocus>
+						<visible>!Integer.IsGreater(Container(4000).NumItems,0)</visible>
+					</control>
+					<control type="textbox">
+						<left>40</left>
+						<top>25</top>
+						<width>690</width>
+						<height>663</height>
```

Comment:
```suggestion
						<height>660</height>
```
Suggest decreasing the height slightly as any lines that go beyond the textbox height peep into the bottom of the textbox.

![image](https://github.com/user-attachments/assets/29da7381-d8f3-4b86-a5e0-15e3a0e3ff2f)

Reply:
Changed.